src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -24,6 +24,31 @@ namespace Microsoft.CodeAnalysis.BuildTasks
     /// </summary>
     public abstract class ManagedCompiler : ManagedToolTask
     {
+        private enum CompilationKind
+        {
+            /// <summary>
+            /// Commpilation occured using the command line tool by normal processes, typically because 
+            /// the customer opt'd out of the compiler server
```

Comment:
```suggestion
            /// the customer opted out of the compiler server
```

Reply:
I contend this isn't necessarily a typo but instead a regional-ism